### DISCUSSION TITLE
New icons missing from website.

### DISCUSSION THREAD

**Discussion by tembbo** [ryanoasis/nerd-fonts#1710] [General]
Previously i requested the bun glyph to be added (#1565), but when i search for bun on the cheat sheet, i cant find it.

![image](https://github.com/user-attachments/assets/5c92f6cb-567f-479c-a7e7-ad9d5e5c4757)

**Comment by Finii** [comment]
Which new icons? Why do you expect to find bun on the cheat sheet? The cheat sheet (of course) represents the latest release, i.e. 3.2.1 from 12/4/2024
There is even no bun icon merged into master iirc...? Therefore my questions above.

**Reply by tembbo** [reply]
Ok, thx. you now when it will be merged

**Reply by Finii** [reply]
https://github.com/ryanoasis/nerd-fonts/discussions/1705

Like end of this month, if all goes smooth.

**Reply by Finii** [reply]
It also depens on upstream
* font logos needs to release first
* commit mono release would be appreciated

**Reply by tembbo** [reply]
thx

**Reply by h2oearth** [reply]
I am looking forward to it :-). Thanks @Finii !
